apps/site/pages/en/learn/modules/how-to-use-streams.md

```diff
@@ -80,29 +80,7 @@ A demonstration of the use of these events can be seen in the following sections
 
 Here's an example of a simple readable stream implementation that generates data dynamically:
 
-```cjs
-const { Readable } = require('node:stream');
-
-class MyStream extends Readable {
-  #count = 0;
-  _read(size) {
-    this.push(':-)');
-    if (++this.#count === 5) {
-      this.push(null);
-    }
-  }
-}
-
-const stream = new MyStream();
-
-stream.on('data', chunk => {
-  console.log(chunk.toString());
-});
-```
-
-```mjs
-import { Readable } from 'node:stream';
-
+```js
```

Comment:
Code vanished?

Reply:
The `#`-prefix (private property) was throwing lint errors in the CJS snippet (`error  Parsing error: Unexpected character '#'`), so I merged the ESM and CJS snippets into a single JS snippet.

The import was removed, since CJS/MJS have different import styles, but I assume it's fine since the snippet right below doesn't include the import.